src/components/I-TeacherSearch/I-TeacherSearch.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import ITab from '../I-Tab/I-Tab';
+import InputField from '../InputField/InputField';
+import Alphabet from '../Alphabet/Alphabet';
+
+import './I-TeacherSearch.css';
+
+type Tab = {
+  label: string,
+  type: Intellect.SearchMode,
+  placeholder?: string
+}
+
+const ITeacherSearch: React.FC = () => {
+  const [activeTab, setActiveTab] = useState('overall' as Intellect.SearchMode);
+  const navigate = useNavigate();
+
+  const tabs = [
+    {label: 'Загальний пошук спiвробiтникiв', type: 'overall', placeholder: 'Введіть ПІБ особи.. (наприклад: Петро Петров Петрович)'},
```

Comment:
Let's provide the same order in an example like ПІБ:
```suggestion
    {label: 'Загальний пошук спiвробiтникiв', type: 'overall', placeholder: 'Введіть ПІБ особи.. (наприклад: Петров Петро Петрович)'},
```